action.yaml

```diff
@@ -73,13 +73,13 @@ runs:
           return issue_nr
     - id: get-repo
       if: ${{ inputs.post-to-issue}}
-      uses: actions/github-script@v5
+      uses: actions/github-script@v7
```

Comment:
This action had breaking changes in the past on major version but I could see nothing in the change log so it should be fine.